react/next/README.md

```diff
@@ -0,0 +1,126 @@
+# Slider Next
+
+## Description
+
+The SliderNext is the upcoming version of VTEX Slider. The main difference is that dots, arrows and slide transitions are now handled by Slider instead of its parent component.
+
+## Table of Contents
+
+- [Usage](#usage)
+  - [Basic](#basic)
+  - [SSR](#ssr)
+  - [Configuration](#configuration)
+- [Upcoming](#upcoming)
+
+## Usage
+
+To import it you can add to you `manifest.json` the following:
+
+```json
+{
+  "dependencies": {
+    "vtex.slider": "0.x"
+  }
+}
+```
+
+And then in your component you can import the components exported from the slider:
+
+```javascript
+import { SliderNext } from 'vtex.slider'
+```
+
+### Basic
+
+The simplistic way of using `SlidexNext` is:
+
+```javascript
+import Component from './**/Component'
+
+...
+
+const items = [...someting...]
+
+<SliderNext>
+  { items.map(item => <Component {...item} />) }
+</SliderNext>
+
+...
+```
+
+Check [configuration](#configuration) to see every prop that `SliderNext` can receive.
+
+### SSR
+
+To use SSR mode you need to define the `responsive` prop that will specify how many items will be presented on each breakpoint. The `responsive` object has the type:
+
+```typescript
+interface responsiveType {
+  /** key is the type name such as mobile, desktop, tablet, ... */
+  [key: string]: {
+    breakpoint: { max: number; min: number }
+    items: number
+  }
+}
+```
+
+So, on your component you will use it like:
+
+```javascript
+  import Component from './**/Component'
+
+  ...
```

Comment:
can you add a real component here? I find it very difficult to understand these docs when they are full of `...`

Reply:
Added a more tangible example